src/design/plone/contenttypes/interfaces/pagina_argomento.py

```diff
@@ -53,14 +52,5 @@ class IPaginaArgomento(model.Schema):
         },
     )
 
-    box_aiuto = RichText(
-        title=_(u"box_aiuto", default=u"Box di aiuto"),
-        required=False,
-        description=_(
-            "box_aiuto_help", default="Eventuali contatti di supporto all'utente."
-        ),
-    )
```

Comment:
Dicevi che sei d'accordo con il rivedere anche i frontend, vero? Perché la pagina argomento per dire è uno dei CT dati per finiti

Reply:
Rigiro la cosa a @giuliaghisini

Reply:
Per me se si capisce sviluppando un ct che c'è qualcosa da sistemare negli altri già fatti per rendere tutto congruente, bisogna farlo. Questa è una mia idea, poi possiamo parlarne..

Reply:
Si, si ci sta: la cosa importante è che non ci disallineiamo come ai primissimi rilasci

Reply:
ok, quindi @cekk box_aiuto è cambiato ovunque in ulteriori informazioni? anche nell'argomento? i ct che lo avevano erano evento, argomento, servizio?